src/core/src/package_managers/YumPackageManager.py

```diff
@@ -360,14 +360,15 @@ def do_processes_require_restart(self):
             self.composite_logger.log_debug(" - Reboot is detected to be required (L1).")
             return True
 
-        # Checking for restart for distro without -r flag such as RHEL 6
-        code, out = self.env_layer.run_command_output(self.needs_restarting, False, False)
-        self.composite_logger.log_debug(" - Code: " + str(code) + ", Output: \n|\t" + "\n|\t".join(out.splitlines()))
-        if len(out.strip()) == 0 and code == 0:
-            self.composite_logger.log_debug(" - Reboot not detected to be required (L2).")
-        else:
-            self.composite_logger.log_debug(" - Reboot is detected to be required (L2).")
-            return True
+        # Checking for restart for distro without -r flag such as RHEL 6 and CentOS 6
+        if '6' == str(self.env_layer.platform.linux_distribution()[1]).split('.')[0]:
```

Comment:
Minor: Flip the order in the equality operator check.. Yoda conditionals do help avoiding accidental assignments but it's not necessary in Python. For readability, the other consideration here, 'blue' == sky_color and sky_color == 'blue' is the common analogy that gets brought up.